stellargraph/layer/graphsage.py

```diff
@@ -119,9 +119,300 @@ def call(self, x, **kwargs):
 
         from_self = K.dot(x[0], self.w_self)
         from_neigh = K.dot(neigh_means, self.w_neigh)
-        total = K.concatenate([from_self, from_neigh], axis=2)
+        h_out = K.concatenate([from_self, from_neigh], axis=2)
 
-        return self.act(total + self.bias if self.has_bias else total)
+        if self.has_bias:
```

Comment:
I see you've replaced the return statement with a more readable one here and in the other two GraphSAGE aggregators, but left the original return statement in HinSAGE's mean aggregator (line 138).

Reply:
I was a bit confused by the return statement and wasn't sure it was doing what I expected. I think the more verbose statement is clearer.

I think graphsage and hinsage aggregators should be synced – I will do this when I implement the pooling aggregators for hinsage :-)

Reply:
Ok, sounds good.